lib/sets.bzl

```diff
@@ -43,6 +43,34 @@ def _precondition_only_sets_or_lists(*args):
             fail("Expected arguments to be depset or list, but found type %s: %r" %
                  (t, a))
 
+def _to_set(a):
```

Comment:
> Isn't this overlapping with #54 which is waiting on final signoff/changes?

Waiting for almost 3 weeks now.

---

> I'd prefer to delete this file, rather than maintain it.

Is there a migration plan for current users? I don't see a deprecation warning in the file.

For example:
1.    add `print("DEPRECATED: ...` at the top of `sets.bzl`
2.    Require an action from existing users:
        * move `sets.bzl` to a `deprecated` subdirectory
        * rename `sets = struct(` to `sets_deprecated = struct(`

I have tried to use an `alias()` to add a deprecation warning, but it doesn't seem to be possible.

Reply:
The first step will be done with https://github.com/bazelbuild/bazel-skylib/pull/70